apps/dashboard/src/main/java/com/akto/listener/KafkaListener.java

```diff
@@ -21,6 +21,12 @@ public void contextInitialized(javax.servlet.ServletContextEvent sce) {
             loggerMaker.infoAndAddToDb("is_kubernetes: true", LogDb.DASHBOARD);
             return;
         }
+
+        if (DashboardMode.isLocalDeployment() && !DashboardMode.isSaasDeployment()) {
+            loggerMaker.infoAndAddToDb("local: true", LogDb.DASHBOARD);
```

Comment:
why this?

Reply:
local deployment does not have kafka dependency. so skipping connecting to kafka in local deployment. Seperate bug fix, irrelevant to current feature.